commands/mr_approvers.go

```diff
@@ -0,0 +1,85 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"glab/internal/git"
+	"glab/internal/manip"
+
+	"github.com/gosuri/uitable"
+	"github.com/logrusorgru/aurora"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+)
+
+var mrApproversCmd = &cobra.Command{
+	Use:     "approvers <id> [flags]",
+	Short:   `List merge requests eligible approvers`,
+	Long:    ``,
+	Aliases: []string{},
+	Args:    cobra.ExactArgs(1),
+	Run:     listMergeRequestEligibleApprovers,
+}
+
+func listMergeRequestEligibleApprovers(cmd *cobra.Command, args []string) {
+	if len(args) > 0 {
+		mergeID := strings.Trim(args[0], " ")
+		//if s, _ := cmd.Flags().GetString("password"); s  {
+		// ToDo:
+		//}
+
+		fmt.Printf("\nListing Merge Request #%v eligible approvers\n", mergeID)
+		gitlabClient, repo := git.InitGitlabClient()
+		if r, _ := cmd.Flags().GetString("repo"); r != "" {
+			repo = r
+		}
+		mrApprovals, _, err := gitlabClient.MergeRequestApprovals.GetApprovalState(repo, manip.StringToInt(mergeID))
+		if err != nil {
+			cmdErr(cmd, args)
+		}
+		if mrApprovals.ApprovalRulesOverwritten {
+			fmt.Println(aurora.Yellow("Approval rules overwritten"))
```

Comment:
Please use github.com/gookit/color instead of aurora... Aurora isn't supported on windows without a tweak. We initially used aurora and had issues with windows

Reply:
That should now be done in 6c44e24 which replaces the previous commits.